.github/workflows/ci.yml

```diff
@@ -19,11 +19,12 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
         cache: 'npm'
-    - run: npm i -g yarn@latest
     - run: npm i -g yarn@1
     - run: npm i -g pnpm@8
     - run: npm ci
     - run: npx playwright install-deps
     - run: npm run build
     - run: npx tsc --noEmit
-    - run: npm run test
+    - run: npm run test -- --project=npm --project=pnpm --project=yarn-classic
```

Comment:
Maybe this is more future proof?

`npm run test -- --grep-invert yarn-classic`
`npm run test -- --grep yarn-berry`

AFAIK grep includes project name.

Reply:
implemented in https://github.com/microsoft/create-playwright/pull/141/commits/97aa4bd1af33355160364b98f6e93a9bdf8b1434